<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProductosList</class>
 <widget class="QDialog" name="frmProductosList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Listado de productos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources.qrc">
    <normaloff>:/assets/productos.png</normaloff>:/assets/productos.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="btnNuevo">
       <property name="text">
        <string>&amp;Nuevo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModificar">
       <property name="text">
        <string>&amp;Modificar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEliminar">
       <property name="text">
        <string>&amp;Eliminar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnActualizarImagen">
       <property name="text">
        <string>&amp;Actualizar imagen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEliminarImagen">
       <property name="text">
        <string>E&amp;liminar imagen</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="imgProducto">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #cccccc;
background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tblProductos">
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>38</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>21</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Categor√≠as</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre del producto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stock</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Referencia</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>P. Compra</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PVP</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/assets/search.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtFiltroSearch"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>frmProductosList</sender>
   <signal>rejected()</signal>
   <receiver>frmProductosList</receiver>
   <slot>closeParent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtFiltroSearch</sender>
   <signal>textChanged(QString)</signal>
   <receiver>frmProductosList</receiver>
   <slot>SearchChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblProductos</sender>
   <signal>currentCellChanged(int,int,int,int)</signal>
   <receiver>frmProductosList</receiver>
   <slot>GridSelectionChanged(int,int,int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNuevo</sender>
   <signal>clicked()</signal>
   <receiver>frmProductosList</receiver>
   <slot>NuevoProducto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnModificar</sender>
   <signal>clicked()</signal>
   <receiver>frmProductosList</receiver>
   <slot>ModificarProducto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEliminar</sender>
   <signal>clicked()</signal>
   <receiver>frmProductosList</receiver>
   <slot>DeleteProductos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblProductos</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>frmProductosList</receiver>
   <slot>ModificarProducto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnActualizarImagen</sender>
   <signal>clicked()</signal>
   <receiver>frmProductosList</receiver>
   <slot>ActualizarImagen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEliminarImagen</sender>
   <signal>clicked()</signal>
   <receiver>frmProductosList</receiver>
   <slot>EliminarImagen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closeParent()</slot>
  <slot>SearchChanged()</slot>
  <slot>GridSelectionChanged(int,int,int,int)</slot>
  <slot>NuevoProducto()</slot>
  <slot>ModificarProducto()</slot>
  <slot>DeleteProductos()</slot>
  <slot>ActualizarImagen()</slot>
  <slot>EliminarImagen()</slot>
 </slots>
</ui>
